stories/Patterns/ListPanel.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { ListPanel } from '../../src';
+import { Meta, StoryObj } from '@storybook/react';
+
+/**
+ * This component has been moved into a pattern. More information about how and when to use this component can be found in the [NHS digital service manual](https://service-manual.nhs.uk/design-system/patterns/a-to-z-page).
+ **/
+const meta: Meta<typeof ListPanel> = {
+  title: 'Patterns/ListPanel',
+  component: ListPanel,
+};
+export default meta;
+type Story = StoryObj<typeof ListPanel>;
+
+export const Standard: Story = {
+  argTypes: {
+    type: { table: { disable: true } },
+  },
+  render: () => (
+    <>
```

Comment:
```suggestion
  render: (args) => (
```

Adding the unused single `args` parameter cleans up the rendering in the `Show Code` section of the generated Docs page

Reply:
Added the arg